lib/rex/post/postgresql/ui/console/command_dispatcher/client.rb

```diff
@@ -0,0 +1,166 @@
+# -*- coding: binary -*-
+
+require 'pathname'
+require 'reline'
+
+module Rex
+  module Post
+    module PostgreSQL
+      module Ui
+
+        ###
+        #
+        # Core PostgreSQL client commands
+        #
+        ###
+        class Console::CommandDispatcher::Client
+
+          include Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher
+
+          # https://www.postgresql.org/docs/current/catalog-pg-database.html
+          # @type [Hash<String, String>]
+          DEFAULT_COLUMN_MAPPINGS = {
+            'oid' => 'Row ID',
+            'datname' => 'Name',
+            'datdba' => 'Owner ID',
+            'encoding' => 'Encoding',
+            'datlocprovider' => 'Locale Provider',
+            'datistemplate' => 'Template?',
+            'datallowconn' => 'Allow Connections?',
+            'datconnlimit' => 'Connection Limit',
+            'datfrozenxid' => 'Frozen XID',
+            'datminmxid' => 'Multixact ID',
+            'dattablespace' => 'Tablespace ID',
+            'datcollate' => 'LC_COLLATE',
+            'datctype' => 'LC_CTYPE',
+            'daticulocale' => 'ICU Locale',
+            'daticurules' => 'ICU Rules',
+            'datcollversion' => 'Collation Version',
+            'datacl' => 'Access Privileges'
+          }.freeze
```

Comment:
Is this just used for getting a human readable version of the column names? just making sure it's not an issue for older/newer postgres version with missing/extra columns

Reply:
It was used for human readable column names when outputting a query. I've removed it instead though as I can see it confusing users:
```
query 'select datname from pg_database'
```
would return `Name` rather than `datname` in the following scenario:
```
PostgreSQL @ 127.0.0.1:5432 (postgres) > query 'select datname from pg_database'
Query
=====

    #  Name
    -  -------
    0  postgres
    1  new_database
    2  template1
    3  template0
    4  second_db
    5  example_user_details
```